x/gov/client/cli/query.go

```diff
@@ -155,10 +155,10 @@ $ %s query gov proposals --page=2 --limit=100
 			}
 
 			if len(res.GetProposals()) == 0 {
-				return fmt.Errorf("no matching proposals found")
+				return fmt.Errorf("no proposals found")
```

Comment:
I think old message is more suited here as there are filters too.
```suggestion
				return fmt.Errorf("no matching proposals found")
```